### DISCUSSION TITLE
How to remove an entry from a table after X time.

### DISCUSSION THREAD

**Discussion by Caleb-T-Owens** [adonisjs/core#1562] [Help]
# How to remove an entry from a table after X amount of time.

I have a situation where I would like to remove an entry from a table after a certain period of time after its creation. Is there some task scheduler that I have missed or something or do I need to make my own solution for this and if so any ideas on the best way to go about it?

**Comment by prashanth1k** [comment]
- Take a look at queuing utilities like Bull - [example schedule](https://github.com/Rocketseat/adonis-bull#scheduled-job).
- Or, schedule a job using `adonis-scheduler` or similar to run a job every `x` min to check "expired" records and remove them. I am not sure whether adonis-scheduler supports v5.
